snow/networking/handler/engine_test.go

```diff
@@ -59,8 +59,8 @@ func TestEngineManager_Get(t *testing.T) {
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
 			e := EngineManager{
-				Avalanche: avalanche,
-				Snowman:   snowman,
+				DAG:   avalanche,
+				Chain: snowman,
```

Comment:
I feel like there really aren't that many places that reference avalanche vs snowman engines... It feels like we should update these var names (and update the one comment).

I didn't find any other unexpected usage